plugins/inputs/elasticsearch_query/elasticsearch_query_test.go

```diff
@@ -3,25 +3,27 @@ package elasticsearch_query
 import (
 	"bufio"
 	"context"
+	"fmt"
 	"os"
 	"strconv"
 	"strings"
-	"sync"
 	"testing"
 	"time"
 
+	"github.com/docker/go-connections/nat"
 	"github.com/google/go-cmp/cmp"
 	"github.com/google/go-cmp/cmp/cmpopts"
 	"github.com/influxdata/telegraf"
 	"github.com/influxdata/telegraf/config"
 	"github.com/influxdata/telegraf/testutil"
 	"github.com/stretchr/testify/require"
+	"github.com/testcontainers/testcontainers-go/wait"
 	elastic5 "gopkg.in/olivere/elastic.v5"
 )
 
-var (
-	testindex = "test-elasticsearch_query-" + strconv.Itoa(int(time.Now().Unix()))
-	setupOnce sync.Once
+const (
+	servicePort = "9200"
```

Comment:
Maybe a comment should be added saying this is a the default elasticsearch port?
Best doc I could find was this: https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-network.html#common-network-settings might be useful to add.